minecraft_servers/keinepixel/manifest.json

```diff
@@ -2,7 +2,19 @@
   "server_name": "keinepixel",
   "nice_name": "KeinePixel",
   "direct_ip": "keinepixel.net",
+  "server_wildcards": [
+    "%.keinepixel.net"
+  ],
   "social": {
     "discord": "https://discord.com/invite/F2h5mDjd6f"
+  },
+  "supported_languages": [
+     "en",
+     "de"
+  ],
+  "location": {
+     "city": "Walldorf",
```

Comment:
Are you sure that your server is located in Walldorf or is it still our default location?